docker/1.4.0/final/Dockerfile.cpu.serving

```diff
@@ -0,0 +1,35 @@
+ARG py_version
+
+FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache mxnet-mkl==1.4.0.post0 \
+                           keras-mxnet==2.2.4.1 \
```

Comment:
dont we need MMS for MXNet inference

Do we need keras-mxnet, onnx etc in this container?

Reply:
I see MMS is coming from: `awsdeeplearningteam/mxnet-model-server`

Reply:
@surajkota sorry for the late response. I believe people use onnx for serving. I'm not too sure about keras-mxnet. What are your thoughts?

Reply:
please check with mxnet on keras-mxnet, its high level api for training.